mono/metadata/w32socket.c

```diff
@@ -742,6 +742,7 @@ get_socket_assembly (void)
 	return domain->socket_assembly;
```

Comment:
I am surprised we need this file at all (is that for debugger?)

Reply:
Yes IIRC